lib/m2ee/client.py

```diff
@@ -36,7 +36,7 @@ def __init__(self, url, password):
         self._url = url
         self._headers = {
             'Content-Type': 'application/json',
-            'X-M2EE-Authentication': b64encode(password)
+            'X-M2EE-Authentication': b64encode(password.encode('utf-8')).decode('ascii'),
```

Comment:
Is there a reason we're decoding the b64 to ASCII instead of UTF-8?

Although, I would imagine that they are equivalent (because base64)

Reply:
+1 here also